src/LaneKeeping/lanekeeping.hpp

```diff
@@ -0,0 +1,89 @@
+//
+// Created by giannishorgos on 04/01/2024.
+//
+
+#pragma once
+
+#include <opencv2/opencv.hpp>
+#include "../laneDetection/detect.hpp"
```

Comment:
I don't know why it works but the folder name starts with a capital L. 
So it should be :
`#include "../LaneDetection/detect.hpp"`

Reply:
Great observation! 
I assume that is not throwing an error because the `hpp` file is included earlier in the program, for example in main.cpp. We have added `include guards` like `#pragma once`  that prevent the file to be imported again. So if the first `#include` is correct, the compiler will not give any attention to wrong includes.

Reply:
Yep that makes sense